src/lucky/content_type_helpers.cr

```diff
@@ -1,19 +1,34 @@
+# These helpers check HTTP headers to determine "content type". Most check the
+# Content-Type header, but some, such as `ajax?` check other fields
+# (`X-Requested-With`).
 module Lucky::ContentTypeHelpers
   abstract def content_type : String
   abstract def headers : HTTP::Headers
 
+  #
```

Comment:
I think a summary on these might look a bit better. Something like: "Check if the request is JSON/AJAZ/XML/etc."